src/components/MultiSelectProfessores.tsx

```
@@ -79,7 +79,7 @@ export function MultiSelectProfessores({
               )
             })
           ) : (
-            <span className="flex items-center gap-2 text-gray-600">Selecione professores <IoIosArrowDown /></span>
+            <span className="flex items-center gap-2 text-gray-600">Professores<IoIosArrowDown /></span>
```

ESLint

[error] Replace `Professores<IoIosArrowDown·/>` with `⏎··············Professores⏎··············<IoIosArrowDown·/>⏎············`

(prettier/prettier)